svm/src/transaction_processor.rs

```diff
@@ -187,6 +180,12 @@ pub struct TransactionBatchProcessor<FG: ForkGraph> {
 
     /// Builtin program ids
     pub builtin_program_ids: RwLock<HashSet<Pubkey>>,
+
+    /// The total stake for the current epoch.
+    pub epoch_total_stake: u64,
+
+    /// The stake for vote accounts for the current epoch.
+    pub epoch_vote_stake: Arc<HashMap<Pubkey, u64>>,
```

Comment:
The `Arc` here is not necessary.

Reply:
This is mainly to resolve this comment: https://github.com/anza-xyz/agave/pull/3671#discussion_r1846311575
For non-agave use case, it allows them to not create and pass the ownership.
It also helps when a new instance of TransactionProcessor is created from an old one (where both instances are in the same epoch).